doc/manual/manual.tex

```diff
@@ -8182,14 +8182,77 @@ \subsubsection{Onset of convection benchmark}
 
 We use bisection to determine $Ra_{cr}$ for specific choices of the domain geometry, keeping the depth $b$ constant and varying the domain width $\lambda$. If the vertical velocity increases from the first to the second timetsp, the system is unstable to convection. Otherwise, it is stable and convection will not occur. Each calculation is terminated after the second timestep. Fig.~\ref{fig:onset-1} shows the numerically-determined threshold for the onset of convection, which can be compared directly with the theoretical prediction (green curve) and Fig.~6.39 of \cite{TS14}. The relative error between the numerically-determined value of $Ra_{cr}$ and the analytic solution are shown in the right panel of Fig.~\ref{fig:onset-1}.
 
-\begin{figure}
+\begin{figure}[h]
  \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/onset-of-convection/racr.png}
  \hfill
  \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/onset-of-convection/racr_error.png}
  \caption{Left: Comparison of numerically-determined and theoretical values for $Ra_{cr}$. Red circles indicate numerical simulations unstable to convection, black circles indicate simulations that are stable. The green dashed curve indicates the theoretical prediction. Right: Relative error in determination of $Ra_{cr}$. The dashed red line indicates the error tolerance used in bisection procedure.}
  \label{fig:onset-1}
 \end{figure}
 
+\subsubsection{The van Keken kinematic corner flow benchmark}
+\label{sec:benchmark-van-keken-subduction}
+\textit{This section is a co-production of Max Rudolph and Jonathan Perry-Houts.}
+
+This benchmark involves a solution to a kinematic corner flow problem that approximates some aspects
+of flow in the mantle wedge. The specific case considered here is `Case 1C' from van Keken et al. \cite{VK08}.
+The flow in this model is driven by a kinematically-prescribed downgoing plate and a kinematically fixed over-riding
+plate. The dip angle of the downgoing plate is 45$^\circ$ and the rate of convergence is 5 cm/yr. The thermal structure of
+the subducing plate is initially described by a halfspace-cooling solution for a plate age of 50 Myr. The over-riding plate is
+50 km thick and has a conductive temperature profile along the right boundary. The top boundary is isothermal. The right boundary and the bottom-right
+corner of the bottom boundary have the natural (stress-free) mechanical boundary condition. Incoming material on the right boundary is isothermal.
+
+In order to treat the kinematically-prescribed regions within the domain, we use a modified version of the `prescribed-velocity'
```

Comment:
Maybe add reference to the section in the manual which describes the prescribed velocity plugin (\ref{sec:prescribed-velocities}). It isn't included in the code; it only exists in the manual right now.